### DISCUSSION TITLE
How to  convert transporters  parameters from literature?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#830] [Questions & Problems] [answered]
Dear to all, I found such a problem in the forum,  and it is also confuses me at present, how to  convert transporters  parameters from literature? MW of hOCT2 is 62.59 kDa, 1 mg protein = 1(mg)/62590*10e9=15.98 pmol protein, so Vmax 11.9 nmol/min/mg = 7.4E-4 nmol/min/pmol. Is this right ?
![image](https://user-images.githubusercontent.com/63133525/89650918-b85e5b00-d8f5-11ea-9dfa-4c68eefe470a.png)

**Comment by prvmalik** [comment, marked as answer]
See issue #343 

The trouble with transporter studies is that they frequently use HEK or MDCK cells that are artificially transfected with certain transporters. So, at present we have no idea how many transporters (e.g. OCT) are present per cell - nor can we be confident that the cells in one experiment have the same transporter expression as the cells in another experiment.

In a typical transporter experiment, drug is incubated with the cells expressing the transporter, and then we check how much of the drug has moved to the intracellular space after a certain amount of time has elapsed. The cells are isolated and lysed (broken apart). The unit: nmol/min/mg protein is measured after the cells are lysed. The amount of drug in the resulting fluid is taken, and then plotted against the total amount of protein in the lysed fluid. This "mg protein" reflects all the protein in the cell. "min" reflects the time of incubation, usually.

Translating this to a PBPK model therefore requires three pieces of information:

1. What is the concentration of the transporter per L of transfected HEK or MDCK cells?
2. What is the concentration of the transporter per L of human renal epithelium?
3. What proportion of the kidney is made up of human renal epithelium?

Basit et al. 2019 have made some progress toward quantifying transporter expression across the whole kidney volume. doi: 10.1124/dmd.119.086579.

p.s. An accepted standard is that there is 200mg protein per mL cells, by the way.

**Comment by StephanSchaller** [comment]
> p.s. An accepted standard is that there is 200mg protein per L cells, by the way.

per liter? I was under the impression it should be around 200-300 mg per mL?

But in general I really don't understand, why (most of the time) these publications don't characterize their systems by providing all of these relevant (reference) information ;-)

**Comment by prvmalik** [comment]
That's right! Good catch

**Comment by Wenjing1018** [comment]
@prvmalik,your reply is helpful for me! 
In cell experiments, the quality of transporters can be measured by the corresponding test kits of transporters. I want to know that when the Vmax is obtained from vitro experiment, it is converted to the units in the software,  is the above calculation correct?

**Comment by prvmalik** [comment]
@Wenjing1018 
Unfortunately the calculation won't work. The Vmax of 11.9 nmol/min/mg protein is per _mg protein in the lysate solution_. This includes all protein in the cells - not just the protein associated with the transporter.

**Comment by Wenjing1018** [comment]
@prvmalik
The test kit can measure the mass of a single transporter. Why  consider the mass of a total protein rather than a transporter?
